nupic/research/frameworks/vernon/README.md

```diff
@@ -0,0 +1,86 @@
+Vernon
+==============
+Vernon is a flexible framework to allow researchers to explore neural network models under a variety of learning paradigms including continuous, meta, and sparse learning. It is designed to be light-weight with a focus on mixin classes. 
```

Comment:
Sparse learning here, compared to meta and continuous learning, implies it is a training paradigm. The term "sparse learning" would be more adequate for something like learning with sparse rewards or sparse observations.

What we aim do is to support different learning paradigms (unsupervised, supervised, continual, meta, etc.) using networks which are biologically inspired, including for example sparse neural networks.